hasura.planx.uk/metadata/tables.yaml

```diff
@@ -1226,6 +1226,37 @@
             }
         template_engine: Kriti
         version: 2
+    - name: update_templated_flow_edits
+      definition:
+        enable_manual: false
+        update:
+          columns:
+            - data
+      retry_conf:
+        interval_sec: 10
+        num_retries: 0
```

Comment:
Again intentionally setting 0 retries here - as this will be a very frequent trigger, think it's safe to assume that if one fails, updates _should_ simply be picked up by the next successful event because it'll always do a full data diff.